Makefile

```diff
@@ -227,9 +229,15 @@ build-ovirt-populator-image:
 push-ovirt-populator-image: build-ovirt-populator-image
 	$(CONTAINER_CMD) push $(OVIRT_POPULATOR_IMAGE)
 
-build-all-images: build-api-image build-controller-image build-validation-image build-operator-image build-virt-v2v-image build-virt-v2v-warm-image build-operator-bundle-image build-operator-index-image build-populator-controller-image build-ovirt-populator-image
+build-openstack-populator-image:
+	$(CONTAINER_CMD) build -f hack/openstack-populator/Containerfile -t $(OPENSTACK_POPULATOR_IMAGE) .
```

Comment:
`hack/openstack-populator/Containerfile` is missing in the PR

Reply:
thanks, added